packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -2,6 +2,64 @@
 title: Okta API Products Release Notes 2021
 ---
 
+## May
+
+### Monthly Release 2021.05.0
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Access-Control-Expose-Headers now available in CORS response](#access-control-expose-headers-now-available-in-cors-response) | May 5, 2021 |
+| [Administrator Roles API third-party admin status update](#administrator-roles-api-third-party-admin-status-update) | May 5, 2021 |
+| [Custom Group Profile Properties now in Early Access](#custom-group-profile-properties-now-in-early-access) | May 5, 2021 |
+| [Domains API is now GA in Preview](#domains-api-is-now-ga-in-preview) | May 5, 2021 |
+| [Domains API supports OAuth 2.0](#domains-api-supports-oauth-2-0) | May 5, 2021 |
+| [Okta Sign-In Widget, version 5.6.0](#okta-sign-in-widget-version-5-6-0) | May 5, 2021 |
+| [Refresh Token Rotation is now GA in Preview](#refresh-token-rotation-is-now-ga-in-preview) | May 5, 2021 |
+| [SAML parameter SessionNotOnOrAfter now available](#saml-parameter-sessionnotonorafter-now-available) | May 5, 2021 |
+| [System Log API SCIM filter expression update](#system-log-api-scim-filter-expression-update) | May 5, 2021 |
+| [Bug fixed in 2021.05.0](#bug-fixed-in-2021-05-0)         | May 5, 2021 |
+
+#### Access-Control-Expose-Headers now available in CORS response
+
+The `Access-Control-Expose-Headers` header is now included in the response to an Okta CORS endpoint call. This header can be accessed by the cross-origin user agent and includes the following possible header names:  `Date`, `Link`, `X-Okta-Edge-Log`, `X-Okta-Request-Id`, `X-Rate-Limit-Limit`, `X-Rate-Limit-Remaining`, and `X-Rate-Limit-Reset`. <!--OKTA-357710-->
```

Comment:
```suggestion
The `Access-Control-Expose-Headers` header is now included in the response to an Okta CORS endpoint call. This header can be accessed by the cross-origin user agent and includes the following possible header names:  `Date`, `Link`, `X-Okta-Edge-Log`, `X-Okta-Request-Id`, `X-Rate-Limit-Limit`, `X-Rate-Limit-Remaining`, and `X-Rate-Limit-Reset`. <!--OKTA-357710-->
```

Would this list of values be easier to read as a bulleted list?

Reply:
Yes, I'll format the values as a bulleted list.